defaults.config.yaml

```diff
@@ -3,10 +3,17 @@ host: 0.0.0.0
 # The port to listen on.
 port: 3000
 # The directory in which to store the LevelDB database.
-databaseDataDirectory: db
-# The directory in which to store uploaded file assets.
-fileDataDirectory: files
+databasePlugin: LevelDB
+databaseConfig:
+  databaseDataDirectory: db
+fileStoragePlugin: awsS3Storage
+fileStorageConfig:
+  bucket: proboci
```

Comment:
Let's not confuse users by showing them two different sets of config when they generally only need one - the default configuration shipped here should be the local file storage option (which works basically without any configuration) and then the README should tell you what to change if you want S3.